<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="widgetGraph" native="true">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>431</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonQuadratic">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>quadratic</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroupRadio</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButtonCubic">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>cubic</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroupRadio</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButtonPlot">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>613</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroupRadio"/>
 </buttongroups>
</ui>
